scripts/pylib/twister/twisterlib/config_parser.py

```diff
@@ -66,6 +66,7 @@ class TwisterConfigParser:
                        "vendor_exclude": {"type": "set"},
                        "extra_sections": {"type": "list", "default": []},
                        "integration_platforms": {"type": "list", "default": []},
+                       "integration_toolchains": {"type": "list", "default": []},
```

Comment:
why no put default as 'zephyr'?

Reply:
because it is not the default for everything, native_sim and unit use the host toolhain..